packages/kit/src/runtime/server/page/render.js

```diff
@@ -146,7 +146,7 @@ export async function render_response({
 
 	let { head, html: body } = rendered;
 
-	const inlined_style = Array.from(styles.values()).join('\n');
+	const rendered_styles = Array.from(styles.values()).join('\n');
```

Comment:
doesn't changing this from `inlined_style` to `rendered_styles` exacerbate the confusion? this contains styles that were imported, _not_ necessarily rendered

Reply:
Maybe I'm still confused. What controls if a style will appear in `node.css` vs `node.styles`? How do the contents of these two variables differ with respect to changing the `css` compiler option?

Reply:
In dev, everything goes into `node.styles`, because that's the only way to avoid FOUC: https://github.com/sveltejs/kit/blob/9bb08d732fcecfca0177e08f0df211cd0065d843/packages/kit/src/core/dev/plugin.js#L96-L102

In build, a stylesheet ends up in `node.styles` if it's below the threshold or we're in AMP mode: https://github.com/sveltejs/kit/blob/9bb08d732fcecfca0177e08f0df211cd0065d843/packages/kit/src/core/build/build_server.js#L240-L252

It will _also_ end up in `node.css`, meaning SvelteKit injects a `<link>`, but with the `disabled` attribute so that the Vite runtime doesn't load stylesheets that are already inlined:

https://github.com/sveltejs/kit/blob/9bb08d732fcecfca0177e08f0df211cd0065d843/packages/kit/src/runtime/server/page/render.js#L228-L230

Reply:
I will freely concede that this is non-obvious and that there are opportunities to both name things less ambiguously and provide clearer comments in the code.